packages/js-dpp/lib/document/stateTransition/DocumentsBatchTransition/DocumentsBatchTransition.js

```diff
@@ -124,6 +126,41 @@ class DocumentsBatchTransition extends AbstractStateTransitionIdentitySigned {
   getModifiedDataIds() {
     return this.getTransitions().map((documentTransition) => documentTransition.getId());
   }
+
+  /**
+   * Returns minimal key security level that can be used to sign this ST
+   *
+   * @override
+   * @return {number}
+   */
+  getKeySecurityLevelRequirement() {
+    const defaultSecurityLevel = IdentityPublicKey.SECURITY_LEVELS.HIGH;
+
+    // Step 1: Get all document types for the ST
+    // Step 2: Get document schema for every type
+    // If schema has security level, use that, if not, use default level
+    // Find the lowest value of all documents
```

Comment:
Find the highest level (lowest value) of all documents ...

Reply:
Fixed